samples/mpsl/timeslot/sysbuild.cmake

```
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+get_property(PM_DOMAINS GLOBAL PROPERTY PM_DOMAINS)
+
+# Include app core image if enabled
+if(SB_CONFIG_SOC_NRF5340_CPUNET)
+  # Get application core board target
+  string(REPLACE "/" ";" split_board_qualifiers "${BOARD_QUALIFIERS}")
+  list(GET split_board_qualifiers 1 target_soc)
+  list(GET split_board_qualifiers 2 target_cpucluster)
+
+  if(DEFINED BOARD_REVISION)
+    set(board_target_appcore "${BOARD}@${BOARD_REVISION}/${target_soc}/cpuapp")
+  else()
+    set(board_target_appcore "${BOARD}/${target_soc}/cpuapp")
+  endif()
+
+  set(target_soc)
+  set(target_cpucluster)
+
+  ExternalZephyrProject_Add(
+    APPLICATION empty_app_core
+    SOURCE_DIR ${ZEPHYR_NRF_MODULE_DIR}/samples/nrf5340/empty_app_core
+    BOARD ${board_target_appcore}
+  )
```

pass revision independently:
```suggestion
    BOARD_REVISION ${BOARD_REVISION}
  )
```

---

Fixed